composer.json

```diff
@@ -1,6 +1,48 @@
 {
-	"require": {
-		"phpunit/phpunit": "3.7.24 - 4",
-		"phing/phing": "dev-master"
-	}
+  "name": "koseven/koseven",
+  "description": "Koseven Framework - PHP-7 compatible replacement for Kohana",
+  "license": "BSD",
+  "keywords": [
+    "kohana",
+    "kohana-php7",
+    "koseven"
+  ],
+  "homepage": "https://koseven.ga/",
+  "authors": [
+    {
+      "name": "Koseven Team",
+      "homepage": "https://koseven.ga/",
+      "role": "developer"
+    },
+    {
+      "name": "Kohana Team",
+      "email": "team@kohanaframework.org",
+      "homepage": "http://kohanaframework.org/team",
+      "role": "developer"
+    }
+  ],
+  "autoload": {
+    "psr-0": {
+      "": [
+        "system/classes"
+      ]
+    }
+  },
+  "support": {
+    "forum": "https://telegram.me/koseven",
+    "issues": "https://github.com/koseven/koseven/issues"
+  },
+  "require": {
+    "php": ">=7.0"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "^6.0",
+    "phing/phing": "^2.16"
+  },
+  "conflict": {
+    "kohana/core": "3.3.*"
+  },
+  "replace": {
+    "kohana/core": "3.3.*"
```

Comment:
This and conflict are important to prevent other packages that rely on kohana core from installing the legacy version e.g. I use [alle/kohana-jsend](https://github.com/alle/kohana-jsend)